lib/kafo/kafo_configure.rb

```diff
@@ -476,14 +476,28 @@ def run_installation
         command = PuppetCommand.new('include kafo_configure', options, puppetconf).command
         log_parser = PuppetLogParser.new
         logger = Logger.new('configure')
-        logger.notice("Starting system configuration")
+
+        start_message = <<-HEREDOC
+Starting system configuration.
+  The total number of configuration tasks may increase during the run.
+  Observe logs or specify --log-level to see individual configuration tasks.
```

Comment:
Looking more closely, the CLI option the user actually wants to use to see more granularity in terminal output is `--verbose-log-level`